README.rst

```diff
@@ -2,9 +2,7 @@
 vulkan-renderer
 ***************
 
-|language| |platforms| |github actions| |readthedocs| |license|
-
-|issues| |last commit| |code size| |contributors| |discord| |downloads|
```

Comment:
Not sure about the contributors, i often use it as an indicator how popular a project is.

Reply:
You can see this on the GitHub page though. I always look at this when I check out new projects.